pkg/util/testing/wrappers.go

```diff
@@ -74,6 +74,29 @@ func MakeWorkload(name, ns string) *WorkloadWrapper {
 	}}
 }
 
+// MakeWorkload creates a wrapper for a Workload with a single
+// pod with a single container.
+func MakeWorkloadWithMorePodSets(name, ns string) *WorkloadWrapper {
```

Comment:
A little confusion about the function name, what do you mean More here, or specify the count directly?